adr/2020-11-06-konstraint-policy-library.md

```diff
@@ -0,0 +1,22 @@
+Alin:
+
+### Integrated Konstraint library to support Common Rego Language definitions.
+
+https://github.com/plexsystems/konstraint/
+
+The purpose of this is to introduce all possible abstractions for evaluating policies - allowing everyone to write the same .rego syntax for both Conftest and Gatekeeper contexts.
+
+Konstraint is used to generate yaml manifests (ConstraintTemplates, Constraint) from source `.rego` files, that get deployed from the `gatekeer-operator-artifacts` chart.
+
+### Usage:
```

Comment:
so no usage, nothing specific to the chosen solution unless it serves in the comparison with other options